java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -30,11 +35,15 @@ public static void main(String[] args) {
         Properties props = KafkaConsumerConfig.createProperties(config);
         int receivedMsgs = 0;
 
-        if (System.getenv("JAEGER_SERVICE_NAME") != null)   {
-            Tracer tracer = Configuration.fromEnv().getTracer();
-            GlobalTracer.registerIfAbsent(tracer);
-
-            props.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, TracingConsumerInterceptor.class.getName());
+        if (System.getenv("JAEGER_SERVICE_NAME") != null) {
```

Comment:
this `if` condition doesn't allow to use OpenTelemetry if you don't set `JAEGER_SERVICE_NAME` env var which of course is wrong.
The `JAEGER_SERVICE_NAME` env var is just for using Jaeger so when OpenTracing is enabled.